server/chat-plugins/hangman.ts

```diff
@@ -307,12 +335,19 @@ export class Hangman extends Rooms.SimpleRoomGame {
 		if (!/[a-zA-Z]/.test(phrase)) throw new Chat.ErrorMessage("Word must contain at least one letter.");
 		let hint;
 		if (params.length > 1) {
-			hint = params.slice(1).join(',').trim();
+			hint = params[1].trim();
```

Comment:
Unsure if I'm a fan of this change, now people can't use commas in their hints. Maybe we can change the param separator to `|` or something?